### DISCUSSION TITLE
Regarding licensing on SixLabors.Fonts" version 1.0.0

### DISCUSSION THREAD

**Discussion by ZubairImran** [ClosedXML/ClosedXML#2418] [Q&A] [answered]
Hi Team,
While i go through dependent assembly license information the SixLabors.Font mentioning split license. Is it OK to use in commercially?
Closed XML using 1.0.0 version and the sixLabor.Fonts is a library as Transitive Package Dependency. If anyone gives some more information would be great help

**Comment by jahav** [comment, marked as answer]
Use version 1.0.

https://github.com/ClosedXML/ClosedXML/issues/2171

**Reply by ZubairImran** [reply]
Thanks @jahav for the prompt reply.
